code/modules/client/preferences.dm

```diff
@@ -1611,6 +1614,11 @@ var/const/MAX_SAVE_SLOTS = 10
 					if(new_relation)
 						nanotrasen_relation = new_relation
 
+				if("synth_manufacturer")
+					var/synth_builder = input(user, "Choose your manufacturer.")  as null|anything in list("Weyland-Yutani", "Borgia Industries", "Hyperdyne Systems", "Lasalle Bionational", "Seegson", "MedTech", "Grant Corporation", "AlphaTech Hardware", "Independent Manufacturer")
```

Comment:
this list should be a global list or define so it can be referenced later for sanitizing